Assets/MRTK/Core/Definitions/Devices/DeviceInputType.cs

```diff
@@ -29,11 +29,22 @@ public enum DeviceInputType
         ButtonPress,
         ButtonTouch,
         ButtonNearTouch,
+        PrimaryButtonPress,
```

Comment:
Adding these to the enum will likely break all existing profiles, since this is serialized into the controller mapping profile and this will change the underlying values of all following entries (like `Trigger` below).

Although I agree that this placement makes the most sense to locate them near related entries, we should add them to the end to maintain existing enum values (or mark each one explicitly, but that could get messy).